### PR TITLE
Update email addresses to use `.com` domain

### PR DESCRIPTION
**Pull request by claireinez**
Apologies for the formatting differences, I think it was down to a merge conflict.

### CODE REVIEW

File: london.html [hunk 1]
```diff
@@ -33,23 +30,11 @@
...
-          <a href="resources.html">Resources</a>
-        </li>
-        <li class="nav-list__list-item">
-          <a href="about.html">About Us</a>
-        </li>
+        <li class="nav-list__list-item"><a href="chapters.html">Chapters</a></li>
+        <li class="nav-list__list-item"><a href="events.html">Events</a></li>
+        <li class="nav-list__logo"><a href="/"><img alt="Node Girls Logo" src="images/ng-logo.svg"></a></li>
+        <li class="nav-list__list-item"><a href="resources.html">Resources</a></li>
+        <li class="nav-list__list-item"><a href="#footer">Contact Us</a></li>
```
Comment (minaorangina) [inline london.html]: This has accidentally changed back to the `Contact Us` link.

File: london.html [hunk 2]
```diff
@@ -60,21 +45,11 @@
...
-            <a href="resources.html">Resources</a>
-          </li>
-          <li>
-            <a href="./about.html">About Us</a>
-          </li>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="chapters.html">Chapters</a></li>
+          <li><a href="events.html">Upcoming Events</a></li>
+          <li><a href="resources.html">Resources</a></li>
+          <li><a href="#footer">Contact Us</a></li>
```
Comment (minaorangina) [inline london.html]: Same as above

**Review by minaorangina — CHANGES_REQUESTED** [review]

File: london.html (hunk 2 above)
Comment (minaorangina) [reply london.html]: Also I've realised this one says "Upcoming Events" when the other navs just say "Events"

**Comment by claireinez** [comment]
Fixed and amended those changes @minaorangina

**Review by minaorangina — APPROVED** [review]

**Pull request merged**